provider/resources.go

```diff
@@ -2897,6 +2926,36 @@ func Provider() tfbridge.ProviderInfo {
 			"google_workstations_workstation_cluster": {Tok: gcpResource(gcpWorkstations, "WorkstationCluster")},
 			"google_workstations_workstation_config":  {Tok: gcpResource(gcpWorkstations, "WorkstationConfig")},
 			"google_workstations_workstation":         {Tok: gcpResource(gcpWorkstations, "Workstation")},
+			"google_workstations_workstation_config_iam_binding": {Tok: gcpResource(gcpWorkstations, "WorkstationConfigIamBinding"),
```

Comment:
Q: Do we capitalise IAM, eg. `WorkstationConfigIAMBinding`? I noticed a mix within this provider `Iam` vs `IAM`. Do we have a convention we should follow and stick with?

Reply:
I think `Iam` is correct so we can distinguish word boundaries when converting to specific language patterns. IIRC there's a list of known accroynms which should convert this to upper case in the right places further down the pipeline.

Reply:
I'm relatively impartial which one we stick with. It'd be hard to enforce a convention at this point given, like you said, it's mixed in this provider already.

Reply:
Though our next major version of this provider should choose one and normalize on it.